build_artifacts/v0/v0.13/v0.13.0/Dockerfile

```diff
@@ -62,6 +62,10 @@ RUN micromamba install -y --name base --file /tmp/$ENV_IN_FILENAME && \
 ARG MAMBA_DOCKERFILE_ACTIVATE=1
 RUN sudo ln -s $(which python3) /usr/bin/python
 
+# Enforce dependencies are all installed from conda-forge
+RUN conda config --remove channels defaults
```

Comment:
Can you update v0/template/Dockerfile as well?

Reply:
good point, updated